<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Live Captions Controller</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Live Captions Controller</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px; font-weight: bold; padding: 10px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="controlGroup">
      <property name="title">
       <string>Controls</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="languageLabel">
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="languageCombo"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="positionLabel">
         <property name="text">
          <string>Position:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="positionCombo"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="opacityLabel">
         <property name="text">
          <string>Opacity: 80%</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="opacitySlider">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start Live Captions</string>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 5px; font-weight: bold;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Stop Live Captions</string>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 5px; font-weight: bold;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="displayGroup">
      <property name="title">
       <string>Captured Text</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="captionDisplay">
         <property name="styleSheet">
          <string notr="true">font-family: Consolas, monospace; font-size: 12px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Ready to start</string>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 5px; background-color: #f0f0f0; border: 1px solid #ccc;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>